components/aggregation/mat/timeScaleXAxis.js

```diff
@@ -5,10 +5,12 @@ import { getDatesBetween } from './computations'
 
 const defaultCount = 20
 
-export function getXAxisTicks (query, count = defaultCount) {
+export function getXAxisTicks (data, count = defaultCount) {
 
-  if (query.axis_x === 'measurement_start_day') {
```

Comment:
We still need this condition, otherwise the charts with `query.axis_x` other than `measurement_start_day` are broken. With moving query to params the changes here will also not be necessary anymore.

Reply:
I'm referring to your question and Arturo's answer about having parameters in the URL. In that case, this change won't be necessary anymore.

> > Was thinking do we need to add the filter parameters in URL also?
> 
> Yeah I think that's a solid plan

Reply:
Hey @majakomel @hellais
Added date and form in the URL of the country page.

Reply:
So, now the `ChartCountry` and `ChartNetwork` components are the same, if everything looks good then I can convert those two back to one file.